lapis/drone.py

```diff
@@ -11,20 +11,24 @@ class ResourcesExceeded(Exception):
 
 
 class Drone(interfaces.Pool):
-    def __init__(self, scheduler, pool_resources: dict, scheduling_duration: float, exclusive: bool=False,
-                 ignore_resources: list=None):
+    def __init__(self, scheduler, pool_resources: dict,
+                 scheduling_duration: float, exclusive: bool = False,
+                 ignore_resources: list = None):
         """
         :param scheduler:
         :param pool_resources:
         :param scheduling_duration:
-        :param exclusive: Determines if the drone is used exclusively by jobs in sequential order
+        :param exclusive: Determines if the drone is used exclusively by jobs
+        in sequential order
```

Comment:
I recommend aligning the body of Sphinx fields internally, not with the key.

```suggestion
                          in sequential order
```